gulp-tasks/test.js

```diff
@@ -0,0 +1,26 @@
+/*
+  Copyright 2014 Google Inc. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+/* eslint-env node */
+
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+
+gulp.task('test:manual', function() {
+  return gulp.src('./test/*.js', {read: false})
+    .pipe(mocha());
```

Comment:
I believe you'd want to add something like

```js
.on('error', function(error) {
  console.error(error);
  process.exit(1);
});
```

to your Gulp stream to fail with a non-zero status if `gulp-mocha` fires an `error` event. That will, in turn, cause the `npm run test` to fail, which is what you'd want.